core/templates/dev/head/forms/formBuilder.js

```diff
@@ -723,6 +723,7 @@ oppia.config(['$provide', function($provide) {
       var _openCustomizationModal = function(
           customizationArgSpecs, attrsCustomizationArgsDict, onSubmitCallback,
           onDismissCallback, refocusFn) {
+        document.execCommand('enableObjectResizing', false, false);
```

Comment:
I initially tried putting this somewhere around 711-718 but it would not work.  This is tested in Chrome, Firefox and Safari on OSX.